src/state/airdrop/hooks.ts

```diff
@@ -75,3 +77,39 @@ export function useClaimCallback(
 
   return { claimCallback }
 }
+
+export interface Questions {
+  id: number
+  title: string
+  content: string
+}
+
+export function useGetQuestions() {
+  var queryData = JSON.stringify({
+    query: `query getKnowledge($filter: kb_filter) {
+      kb(filter: $filter) {
+          id
+          title
+          content
+      }
+  }`,
+    variables: { filter: { category: { _eq: 'Airdrop' } } }
+  })
+
+  const headers = {
+    'Content-Type': 'application/json'
+  }
+
+  return useQuery('getQuestions', async () => {
+    const response = await axios.post('https://p7gm7mqi.directus.app/graphql', queryData, { headers: headers })
```

Comment:
please move this url to `src/constants/index.ts` on top

Reply:
URL moved to constant and lint errors have been solved